Source/src/components/animation/AnimController.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "Math.h"
```

Comment:
If this is external, let's use angular bracets.
```suggestion
#include <Math.h>
```